src/pages/tBTC/Bridge/components/BridgeProcessCardTitle.tsx

```diff
@@ -1,50 +1,81 @@
-import { FC } from "react"
-import { HiArrowNarrowLeft } from "react-icons/all"
-import { Icon, Stack, LabelSm } from "@threshold-network/components"
-import { tBTCFillBlack } from "../../../../static/icons/tBTCFillBlack"
-import { BridgeProcess, MintingStep } from "../../../../types/tbtc"
-import { TBTCText } from "../../../../components/tBTC"
+import { ComponentProps, FC, ReactNode } from "react"
+import {
+  BodyLg,
+  BodySm,
+  Box,
+  Flex,
+  HStack,
+  Icon,
+  VisuallyHidden,
+  VStack,
+} from "@threshold-network/components"
+import { MintingStep, MintingSteps as mintingSteps } from "../../../../types"
+import { FaChevronLeft as ChevronLeftIcon } from "react-icons/fa"
 
-const bridgeProcessToText: Record<BridgeProcess, string> = {
-  mint: "minting process",
-  unmint: "unminting process",
-}
-
-type CommonProps = {
-  bridgeProcess?: BridgeProcess
-}
-
-type ConditionalProps =
-  | {
-      previousStep?: MintingStep
-      onPreviousStepClick: (previosuStep: MintingStep) => void
-    }
-  | {
-      previousStep?: never
-      onPreviousStepClick?: never
-    }
-
-type Props = CommonProps & ConditionalProps
+const TOTAL_STEPS = mintingSteps.length - 1
+// -1 because we don't count the success step
 
-export const BridgeProcessCardTitle: FC<Props> = ({
-  previousStep,
+export const BridgeProcessCardTitle: FC<
+  {
+    number: number
```

Comment:
I feel like I've added this comment before but somehow I can't find it 🤔 Maybe it was deleted due to refresh or something, so I'll write it again:

Can we change the name of `number` property? I feel like it's too generic and doesn't say anything about what that property is doing and what this number is for. What about `stepNumer` or just `step`, since it is displayed next to `TOTAL_STEPS` number?

Reply:
Ref commit: 6e2f421a